lib/pages/new_private_chat/widget/expansion_list.dart

```diff
@@ -177,9 +169,7 @@ class ExpansionList extends StatelessWidget {
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          _NewGroupButton(
-            onPressed: goToNewGroupChat,
-          ),
+          ..._buildResponsiveButtons(context),
           _GetHelpButton(),
```

Comment:
IMO, please remove it, no function for it now

Reply:
button doesn't appear on web so IMO it's cleaner than writing this 6 times
```dart
    if (getIt.get<ResponsiveUtils>().isSingleColumnLayout(context)) ;
      _NewGroupButton(
        onPressed: goToNewGroupChat,
      ),
```

Reply:
no, I mean `_GetHelpButton(),`